lib/github-glue.ts

```diff
@@ -227,20 +226,20 @@ export class GitHubGlue {
      * @param {string} comment the comment to add
      * @returns the comment ID and the URL to the added comment
      */
-    public async addPRCommentReply(pullRequestURL: string, id: number,
+    public async addPRCommentReply(pullRequest: pullRequestKeyInfo, id: number,
                                    comment: string):
         Promise<{id: number, url: string}> {
-        const [owner, repo, nr] =
-            GitGitGadget.parsePullRequestURL(pullRequestURL);
-        await this.ensureAuthenticated(owner);
+        const prKey = getPullRequestKey(pullRequest);
+
+        await this.ensureAuthenticated(prKey.owner);
 
         const status = await this.client.rest.pulls.createReplyForReviewComment(
             {
                 body: comment,
                 comment_id: id,
-                owner,
-                pull_number: nr,
-                repo,
+                owner: prKey.owner,
+                pull_number: prKey.number,
```

Comment:
If we called the attribute `prKey.pull_number`, we could use the `...prKey` syntax to DRY up the code even further (not only here, but elsewhere, too). Even if we don't, `pull_number: prKey.number, ...prKey` might work.

Reply:
Great idea!  Felt awkward using a type as a name anyway.

I did not use `...prKey` when `issue_number` was part of the request.  Concerned about future rejection of unexpected parameters (pull_number).  Should this be a concern?

Reply:
> I did not use ...prKey when issue_number was part of the request.

Makes sense!